jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/JmxScraper.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxscraper;
+
+import io.opentelemetry.contrib.jmxscraper.config.ConfigurationException;
+import io.opentelemetry.contrib.jmxscraper.config.JmxScraperConfig;
+import io.opentelemetry.contrib.jmxscraper.config.JmxScraperConfigFactory;
+import io.opentelemetry.contrib.jmxscraper.jmx.JmxClient;
+import java.net.MalformedURLException;
+import java.util.Arrays;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+public class JmxScraper {
+  private static final Logger logger = Logger.getLogger(JmxScraper.class.getName());
+  private final ScheduledExecutorService exec = Executors.newSingleThreadScheduledExecutor();
+  private final JmxScraperConfig config;
+
+  JmxScraper(JmxScraperConfig config) {
+    this.config = config;
+
+    try {
+      @SuppressWarnings("unused") // TODO: Temporary
+      JmxClient jmxClient = new JmxClient(config);
+    } catch (MalformedURLException e) {
+      throw new ConfigurationException("Malformed serviceUrl: ", e);
+    }
+  }
+
+  @SuppressWarnings("FutureReturnValueIgnored") // TODO: Temporary
+  private void start() {
+    exec.scheduleWithFixedDelay(
+        () -> {
+          logger.fine("JMX scraping triggered");
+          //            try {
+          //              runner.run();
+          //            } catch (Throwable e) {
+          //              logger.log(Level.SEVERE, "Error gathering JMX metrics", e);
+          //            }
+        },
+        0,
+        config.getIntervalMilliseconds(),
+        TimeUnit.MILLISECONDS);
+    logger.info("JMX scraping started");
+  }
+
+  private void shutdown() {
+    logger.info("Shutting down JmxScraper and exporting final metrics.");
+    exec.shutdown();
```

Comment:
[minor] even if there is only one task executed, making sure that the pool is properly shut down with code similar to https://github.com/elastic/apm-agent-java/blob/48d2a231e604414ccb4385792a8c5e364567dcd9/apm-agent-core/src/main/java/co/elastic/apm/agent/util/ExecutorUtils.java#L239